### PR TITLE
Improve focus functionality of EntityReferenceAutocomplete

### PR DESCRIPTION
**Pull request by porkloin**
## Issue

https://github.com/jsdrupal/drupal-admin-ui/issues/266

## Screenshot / UI changes

N/A

## Testing instructions

Visit /node/add/recipe – click any of the autocomplete entity reference fields (Recipe category, or Tags) and begin typing. Focus should remain on the text field consistently while typing.

### CODE REVIEW

File: src/components/02_atoms/Widgets/EntityReferenceAutocomplete.js [hunk 1]
```diff
@@ -205,7 +205,6 @@ class EntityReferenceAutocomplete extends React.Component {
               fullWidth: true,
               label: this.props.label,
               InputProps: getInputProps({
-                disabled: this.state.loading,
```
Comment (mattgrill) [inline src/components/02_atoms/Widgets/EntityReferenceAutocomplete.js]: There is perhaps a bit of nuance here. If the request is inflight, do we want to send another request on every keystroke?

Comment (porkloin) [reply src/components/02_atoms/Widgets/EntityReferenceAutocomplete.js]: Yeah, that makes a lot of sense. I'll take another stab at it and post another commit soon.

Comment (lauriii) [reply src/components/02_atoms/Widgets/EntityReferenceAutocomplete.js]: >  If the request is inflight, do we want to send another request on every keystroke?

Why wouldn't we send another request on every keystroke?

**Comment by lauriii** [comment]
Currently the suggestions paper is closed after a keystroke. To make the interaction feel smoother, I think that the suggestions should be visible all the time.

entity_reference_widget_modal_flicker

File: src/components/02_atoms/Widgets/EntityReferenceAutocomplete.js (hunk 1 above)
Comment (mattgrill) [reply src/components/02_atoms/Widgets/EntityReferenceAutocomplete.js]: I think in general it's a bad idea to send a network request on each keystroke. That's potentially hundreds of requests for a single search. Perhaps adding a slight delay to attempt to guess when the user is done typing and issue the request then?

Comment (lauriii) [reply src/components/02_atoms/Widgets/EntityReferenceAutocomplete.js]: Should we consider that if we run into any problems? I suggest that, for now, we do the request on every keystroke and improve it later if that is necessary.

Comment (dawehner) [reply src/components/02_atoms/Widgets/EntityReferenceAutocomplete.js]: Drupal core ran with that strategy for a long time. I think that seems fine for itself

**Pull request merged**